crates/red_knot_test/src/matcher.rs

```diff
@@ -180,6 +180,15 @@ where
     }
 }
 
+/// Discard `@Todo`-type metadata from expected types, which is not available
+/// when running in release mode.
+#[cfg(not(debug_assertions))]
+fn discard_todo_metadata(ty: &str) -> std::borrow::Cow<'_, str> {
+    regex::Regex::new(r"@Todo\([^)]*\)")
```

Comment:
I was just about to write that I didn't bother doing that because it's just a test and we only run this when running tests in release mode, but okay :smile: